cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -310,6 +310,35 @@ def test_revert_active_snapshot(full_test_vm, provider, soft_assert, register_ev
                            active_snapshot=True)
 
 
+@pytest.mark.rhv2
+@pytest.mark.provider([RHEVMProvider], override=True)
+@pytest.mark.meta(automates=[BZ(1552732)])
+def test_revert_to_active_vm(small_test_vm, provider):
+    """
+    Test that it's not possible to revert to "Active VM" on RHV.
+
+    Bugzilla:
+        1552732
+
+    Metadata:
+        test_flag: snapshot
+
+    Polarion:
+        assignee: anikifor
+        initialEstimate: 1/4h
+        casecomponent: Infra
+    """
+    snapshot = new_snapshot(small_test_vm, has_name=False)
+    snapshot.create()
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_OFF, cancel=False)
+    small_test_vm.wait_for_vm_state_change(desired_state=small_test_vm.STATE_OFF)
+    snapshot.revert_to()
+    provider.browser.refresh()
+    view = navigate_to(small_test_vm, 'SnapshotsAll')
```

Comment:
may be `force` navigation help here to avoid browser refresh.

Reply:
@digitronik thanks! I tried locally and it does what I need :)